powerline/segments/common/net.py

```diff
@@ -74,6 +74,7 @@ def internal_ip(pl, interface='auto', ipv=4):
 	_interface_starts = {
 		'eth':      10,  # Regular ethernet adapters         : eth1
 		'enp':      10,  # Regular ethernet adapters, Gentoo : enp2s0
+        'en':       10,  # OS X                              : en0 
```

Comment:
Wondered the same when I made the pull request, but on vim looks ok (expandtabs deactivated), and so does in Sublime.

Anyway, it's inside a dict definition, so identation shouldn't really matter, and it's working on my system.